nixos/tests/nextcloud/with-declarative-redis-and-secrets.nix

```diff
@@ -93,7 +92,8 @@ runTest (
             {
               "redis": {
                 "password": "secret"
-              }
+              },
+              "dbpass": "${databasePassword}"
```

Comment:
Hmm... still not so sure if I like this approach now that we have LoadCredential in place. Sure, with stuff like sops-nix you can do templating of these secret files, but it's still a little weird.

Can't we do `dbpassFile` etc. in `settings` directly and drop the key when generating the final override.config.php

cc @networkException who authored the loadcredential part for opinions.

Reply:
> I'm very much in favor of keeping dbpassFile, dropping it feels like a step back.

I agree on that

> That said it looks like I've actually missed cfg.secretFile entirely in my LoadCredential bringup (probably because it overlaps with cfg.config.objectstore.s3.secretFile 🙃), so the state of that isn't ideal as a whole currently.

Heh, me too. Would you be up for adding this as well to the LoadCredential code @networkException ? :) 

@onny do you think it'd be feasible to leave out `dbpass` and add `dbpassFile` somewhere (maybe even in settings, or in something like `services.nextcloud.secrets` - so that we'd have `services.nextcloud.secrets.dbpassFile`).
The latter only makes sense though if we do this for _all_ secrets.

Reply:
> Heh, me too. Would you be up for adding this as well to the LoadCredential code @networkException ? :)

Added in https://github.com/NixOS/nixpkgs/pull/405970